### DISCUSSION TITLE
overview of the zustand API

### DISCUSSION THREAD

**Discussion by alphabetabc** [pmndrs/zustand#2385] [Q&A] [answered]
There is no API overview in the documentation, which is not friendly to beginners. Here is a simple list of APIs 

![image](https://github.com/pmndrs/zustand/assets/29876424/eaf38772-5661-436b-b0c7-7ef165642602)

**Comment by alphabetabc** [comment]
- API@v4.5.2
  - index
    - create
    - createStore
    - useStore
  - zustand/react/shallow
    - useShallow
  - zustand/vanilla
    - createStore
  - zustand/middleware
    - combine
    - createJSONStorage
    - devtools
    - persist
    - redux
    - subscribeWithSelector
  - zustand/middleware/immer
    - immer
  - zustand/shallow
    - shallow
  - zustand/vanilla/shallow
    - shallow
  - zustand/react/shallow
    - useShallow
  - zustand/traditional
    - createWithEqualityFn
    - useStoreWithEqualityFn
  - ~~zustand/context ---> @deprecated~~

**Comment by dai-shi** [comment, marked as answer]
It has been discussed in several places.
#2327 #206
